### PR TITLE
Ensure we are deploying the right project

### PR DESCRIPTION
**Pull request by lukebjerring**

### CODE REVIEW

File: git/git-deploy [hunk 1]
```diff
@@ -36,7 +36,7 @@ then
   PROMOTE="--promote"
 fi
 
-COMMAND="gcloud app deploy ${PROMOTE} --version=${VERSION} ${WPTD_PATH}/webapp"
+COMMAND="gcloud config set project wptdashboard && gcloud app deploy ${PROMOTE} --version=${VERSION} ${WPTD_PATH}/webapp"
```
Comment (rwaldron) [inline git/git-deploy]:
```
$ git/git-deploy

[  2018-02-13 14:49:00  INFO  ]  Deploy command:
gcloud config set project wptdashboard && gcloud app deploy --no-promote --version=w3c-lukebjerring-patch-1 /Users/rwaldron/clonez/wptdashboard/webapp

[  2018-02-13 14:49:00  WARN  ]  Execute? (Y/n)
y
[  2018-02-13 14:49:06  INFO  ]  Executing...
ERROR: (gcloud.config.set) unrecognized arguments:
  &&
  gcloud
  app
  deploy
  --no-promote (did you mean '--format'?)
  --version=w3c-lukebjerring-patch-1 (did you mean '--verbosity'?)
  /Users/rwaldron/clonez/wptdashboard/webapp
```

Comment (lukebjerring) [reply git/git-deploy]: hmmm, && meaning is being dropped. Fixed with an eval (and tested locally).

Comment (rwaldron) [reply git/git-deploy]: With a clean checkout of your branch: 

```
rwaldron in ~/clonez/wptdashboard on lukebjerring-patch-1
$ git/git-deploy

[  2018-02-13 17:15:10  INFO  ]  Deploy command:
gcloud config set project wptdashboard && gcloud app deploy --no-promote --version=w3c-lukebjerring-patch-1 /Users/rwaldron/clonez/wptdashboard/webapp

[  2018-02-13 17:15:10  WARN  ]  Execute? (Y/n)
y
[  2018-02-13 17:15:23  INFO  ]  Executing...
Updated property [core/project].


Updates are available for some Cloud SDK components.  To install them,
please run:
  $ gcloud components update


Services to deploy:

descriptor:      [/Users/rwaldron/clonez/wptdashboard/webapp/app.yaml]
source:          [/Users/rwaldron/clonez/wptdashboard/webapp]
target project:  [wptdashboard]
target service:  [default]
target version:  [w3c-lukebjerring-patch-1]
target url:      [https://w3c-lukebjerring-patch-1-dot-wptdashboard.appspot.com]


     (add --promote if you also want to make this service available from
     [https://wptdashboard.appspot.com])

Do you want to continue (Y/n)?
Beginning deployment of service [default]...
Some files were skipped. Pass `--verbosity=info` to see which ones.
You may also view the gcloud log file, found at
[/Users/rwaldron/.config/gcloud/logs/2018.02.13/17.15.25.644282.log].
╔════════════════════════════════════════════════════════════╗
╠═ Uploading 0 files to Google Cloud Storage                ═╣
╚════════════════════════════════════════════════════════════╝
File upload done.
Updating service [default]...failed.
ERROR: (gcloud.app.deploy) Error Response: [9] Deployment contains files that cannot be compiled: Compile failed:
/work_dir/anomaly_handler.go:34: undefined: metrics.GetDatastoreKindName
/work_dir/anomaly_handler.go:35: undefined: metrics.PassRateMetadata
/work_dir/anomaly_handler.go:37: undefined: metrics.PassRateMetadata
/work_dir/anomaly_handler.go:69: undefined: metrics.GetDatastoreKindName
/work_dir/anomaly_handler.go:70: undefined: metrics.FailuresMetadata
/work_dir/anomaly_handler.go:74: undefined: metrics.FailuresMetadata
/work_dir/interop_handler.go:20: undefined: metrics.GetDatastoreKindName
/work_dir/interop_handler.go:20: undefined: metrics.PassRateMetadata
/work_dir/interop_handler.go:23: undefined: metrics.PassRateMetadata
2018/02/13 14:15:56 go-app-builder: build timing: 16×compile (9.164s total), 0×link (0s total)
2018/02/13 14:15:56 go-app-builder: failed running compile: exit status 2
```

**Comment by lukebjerring** [comment]
This seems like an issue with your local $GOPATH (probably the same as #456), as I don't think `/work_dir/` is part of the repo? Have you sym-linked something in your src directory? Have you tried the same thing from a cloud shell?

**Comment by rwaldron** [comment]
> This seems like an issue with your local $GOPATH (probably the same as #456), as I don't think /work_dir/ is part of the repo? 

Correct, `work_dir` appears to be a temp directory name convention used throughout  is the default path that 

> Have you sym-linked something in your src directory? 

No.

I figured out the problem this morning: 

1. For local development, the go deps are fetched and installed within the docker instance
2. The meager tests written for anomaly and interop (in main_test.go) are run in the docker instance
3. The site is run from the docker instance for development. 
4. git/git-deploy does not get executed from the docker instance, so unless the developer has explicitly run `go get github.com/w3c/wptdashboard/metrics` from their local (not in docker) environment, any deps that were installed in the docker instance will be missing from the deployment (which leads to the error I've encountered). 

I am able to reproduce this consistently by running `go get github.com/w3c/wptdashboard/metrics`, deploying, then `go clean ...` and deploying, which results in an erroneous deployment.

**Pull request closed without merging**